src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/attachscanneddocs/AttachScannedDocsAboutToSubmitHandler.java

```diff
@@ -38,24 +39,28 @@ public PreSubmitCallbackResponse<SscsCaseData> handle(CallbackType callbackType,
         setHasUnprocessedAudioVideoEvidenceFlag(sscsCaseData);
 
         if (deletedRedactedDocEnabled) {
-            var scannedDocuments = callback.getCaseDetails().getCaseData().getScannedDocuments();
+            try {
+                var scannedDocuments = callback.getCaseDetails().getCaseData().getScannedDocuments();
 
-            callback.getCaseDetailsBefore().ifPresent(
-                    sscsCaseDataCaseDetailsBefore -> {
-                        sscsCaseDataCaseDetailsBefore.getCaseData().getScannedDocuments()
-                                .stream()
-                                .filter(scannedDocumentBefore -> scannedDocumentBefore.getValue().getEditedUrl() != null)
-                                .forEach(scannedDocumentBefore -> {
-                                    scannedDocuments
-                                            .stream()
-                                            .forEach(scannedDocument -> {
-                                                if (compare(scannedDocumentBefore, scannedDocument)) {
-                                                    scannedDocument.getValue().setEditedUrl(scannedDocumentBefore.getValue().getEditedUrl());
-                                                }
-                                            });
-                                });
+                callback.getCaseDetailsBefore().ifPresent(
+                        sscsCaseDataCaseDetailsBefore -> {
+                            sscsCaseDataCaseDetailsBefore.getCaseData().getScannedDocuments()
+                                    .stream()
+                                    .filter(scannedDocumentBefore -> scannedDocumentBefore.getValue().getEditedUrl() != null)
+                                    .forEach(scannedDocumentBefore -> {
+                                        scannedDocuments
+                                                .stream()
+                                                .forEach(scannedDocument -> {
+                                                    if (compare(scannedDocumentBefore, scannedDocument)) {
+                                                        scannedDocument.getValue().setEditedUrl(scannedDocumentBefore.getValue().getEditedUrl());
+                                                    }
+                                                });
+                                    });
 
-                    });
+                        });
+            } catch (NullPointerException e) {
```

Comment:
why do we want to silently handle an NPE? if there's a possibility of an NPE then we should add a null check